app/controllers/subscriptions_controller.rb

```diff
@@ -15,6 +15,7 @@ def new
 
     if params[:email]
       @email = params[:email]
+      @subscription = Subscription.find_by(email: @email)
```

Comment:
> However it's worth asking ourselves if we really need this element. As you just said, "People will most likely be coming from an alert email if they see that", so why show it again?

Hmmm. I do think it's helpful to have it there on the page with the information about what the subscription does for you—but, again it feels like over optimising at this point, I'm sure we'll make another changes as we iterate and we add it if it feels necessary.

I’m going to cut it.